### DISCUSSION TITLE
Code Design Question - Get & Set Methods

### DISCUSSION THREAD

**Discussion by LercDsgn** [corpengine/corpengine#25] [Questions]
### Which form of code do you prefer?

* ![image](https://user-images.githubusercontent.com/75680333/151659410-9631dbfd-2c5e-459e-9b18-285180780d22.png)
* ![image](https://user-images.githubusercontent.com/75680333/151659359-4b542d0a-55eb-49d3-b8a1-5087a7be2c5e.png)
* ![image](https://user-images.githubusercontent.com/75680333/151659384-0bd74502-7220-4f98-b191-21247ba72eea.png)

**Comment by polaritynord** [comment]
Definitely the first.
Why? Because I don't really like several jobs put in a single method. It also affects the readability in my opinion.

**Reply by LercDsgn** [reply]
I think the version function kind of conflicts with this choice. Should we revert it back to its original class form?

**Reply by polaritynord** [reply]
maybe
